source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,275 @@
+.. _realm-sync-migrate-modes:
+
+=========================
+Migrate Device Sync Modes
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+Atlas App Services Apps can have one of two Device Sync modes: Partition-Based Sync
+and Flexible Sync. Partition-Based Sync is an older mode and you should consider
+migrating to Flexible Sync.
+
+Migrating an App that uses Partition-Based Sync to Flexible Sync is an automatic
+process. Other than upgrading the SDK version, migrating Sync modes doesn't require 
+any changes to your client app code.
+
+Migration results in Partition-Based Sync clients communicating with a
+Flexible Sync backend. We recommend eventually migrating your client apps to also
+use Flexible Sync instead of Partition-Based Sync.
+
+Requirements
+------------
+
+App Services App:
+
+- Currently use Partition-Based Sync
+
+Linked Atlas Cluster:
+
+- Use MongoDB 5.0 or later for Flexible Sync compatibility
+- Have storage auto-scaling enabled
+- Be an M10+ Cluster Tier
+- Have time-based oplog setting enabled
+
+Client apps:
+
+- Use ``realm-core`` vTBD+, which means these SDK versions: TBD
```

Comment:
`realm-core` version possibly 13.8.0.

Reply:
Update before deploying!

Reply:
Thanks for the reminder! We still don't know what Core version (or SDK versions) will be required yet.